tsconfig.json

```diff
@@ -31,11 +31,22 @@
       "react"
     ]
   },
-  "include": [],
+  "include": [
+    "e2e/__helpers__",
+    "e2e/__serializers__",
+    "e2e/__tests__",
+    "scripts/",
+    "src",
+    "presets",
+    "./*.js"
+  ],
   "exclude": [
     "e2e/__cases__",
     "e2e/__templates__",
     "e2e/__workdir_synlink__",
-    "node_modules"
+    "e2e/__external-repos__",
```

Comment:
does it work if we replace these changes for `exclude` by using only `.eslintignore` ?

Reply:
@ahnpnl I can try!